lib/gollum/templates/layout.mustache

```diff
@@ -56,8 +58,26 @@
   {{/mathjax_config}}
   <script defer src="{{mathjax_js}}"></script>
   {{/mathjax}}
+  
+  {{#mermaid}}
+  <script>
+  var mermaid_conf = {
+    startOnLoad: true,
+    securityLevel: 'strict'
+  };
+  </script>
+  {{/mermaid}}
+  
   {{#js}}<script type="text/javascript" src="{{custom_js}}"></script>{{/js}}
 
+  {{#mermaid}}
+  {{#sprockets_javascript_tag}}mermaid{{/sprockets_javascript_tag}}
+  <script>
+    console.log(mermaid_conf)
```

Comment:
leftover debug statement?

Reply:
Yes, I left this in because I was still trying to get the tests to succeed. :) Thanks for marking it!